object/check.go

```diff
@@ -483,7 +483,17 @@ func CheckUserPermission(requestUserId, userId string, strict bool, lang string)
 
 	hasPermission := false
 	if strings.HasPrefix(requestUserId, "app/") {
-		hasPermission = true
+		requestApp, err := GetApplication(ctx, fmt.Sprintf("admin/%s", strings.Split(requestUserId, "/")[1]))
+		if err != nil {
+			return false, err
+		}
+		if requestApp == nil {
+			return false, fmt.Errorf(i18n.Translate(lang, "check:Session outdated, please login again"))
```

Comment:
mb application not found?

Reply:
it's copied from block for user for sameness of error in func. It's means that you use session for app that was deleted or renamed